xgi/convert.py

```diff
@@ -71,23 +72,39 @@ def convert_to_hypergraph(data, create_using=None):
     if data is None:
         return empty_hypergraph(create_using)
 
-    elif isinstance(data, Hypergraph):
+    elif isinstance(data, Hypergraph) and not isinstance(data, SimplicialComplex):
+        H = empty_simplicial_complex(create_using)
         H = empty_hypergraph(create_using)
```

Comment:
The first line seems to be unnecessary, no? H gets overwritten in the next line by a hypergraph.